modulus/models/diffusion/song_unet.py

```diff
@@ -169,6 +171,17 @@ def __init__(
             )
 
         super().__init__(meta=MetaData())
+        if gridtype[:9] == "learnable":
+            pos_dim = int(gridtype[10:])
+            print("learnable %s positional embedding" % gridtype[10:])
```

Comment:
These type of print statements should not be in the network architecture. Consider adding that to the training script instead.